tftb/processing/base.py

```diff
@@ -11,7 +11,7 @@
 """
 
 import numpy as np
-from scipy.signal import hamming
+from scipy.signal.windows import hamming
```

Comment:
Is this needed? AFAIK the following
```python
from scipy.signal import hamming
```

works fine.

Reply:
Yes, both work fine, so I have reverted to the original import.
My IDE was marking the original version with an error and therefore I changed it.
On closer examination I found that hamming is actually located in scipy.signal.windows. However, Scipy provides a convenience function so that the import from scipy.signal works.